### ISSUE TITLE
Replies not being discarded

### ISSUE THREAD

**Issue by dufia** [superscriptjs/superscript#389]
```
+ Hi *
- Hey!
- Hi, good to see you!
- Hey, welcome back!
- Hello to you too!
```

In this simple example if you type "Hi" again and again you can get the same replay even three times in a row. This isn't working as expected I presume.

**Comment by LucienBrule** [comment]
I don't believe it's super clear in the documentation yet, but on further inspection, any gambits (reply pairings of +-) at the root level in superscript are kept (with {keep}) implicitly and are placed in the random topic. To discard the a response one needs to create a topic (without {keep}) then place the gambit inside (with {ordered}). Here's an example I got to work just now:
```
> topic greeting (Hello, name)
  + {ordered} Hello
  - Howdy!
  - Bounjour.
  - Allo
  - Hola
  - Suh
  - We've said hello already
< topic

> topic random
  + Hello
  - Switching to greeting topic {topic=greeting} ^respond("greeting")
< topic
```

So here you'll get "Switching to random topic Howdy!" from random.Hello, followed by all of the responses in Greeting.Hello on subsequent "Hello"s submitted from a client.

I hope that helps!
